### PR TITLE
Form - update description for the floating labels feature

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

File: api-reference/10 UI Components/dxForm/1 Configuration/showColonAfterLabel.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ default: true, false (Material)
 ---
 ---
 ##### shortDescription
-Specifies whether or not a colon is displayed at the end of form labels.
+Specifies whether or not a colon is displayed at the end of form labels. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels.
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxForm/1 Configuration/showColonAfterLabel.md]: In the editor's labels, the option will have no effect and the colon will not be displayed

File: api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/showColon.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ default: from showColonAfterLabel
 ---
 ---
 ##### shortDescription
-Specifies whether or not a colon is displayed at the end of the current label.
+Specifies whether or not a colon is displayed at the end of the current label. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels.
```
Comment (EugeniyKiyashko) [inline api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/showColon.md]: In the editor's labels, the option will have no effect and the colon will not be displayed

File: api-reference/10 UI Components/dxForm/1 Configuration/alignItemLabels.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-Specifies whether all item labels are aligned. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"*.
+Specifies whether all item labels are aligned. Applies only if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxForm/1 Configuration/alignItemLabels.md]: "if ... [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the labels" means "if labels are visible", which is kind of an unnecessary thing to say. It's obvious that if labels are hidden, they cannot be aligned.

File: api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/visible.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 id: dxFormSimpleItem.label.visible
 type: Boolean
-default: true
+default: true if the [labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"*. false otherwise.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/visible.md]: This cannot be done in docs, you should use doccomments.

File: api-reference/10 UI Components/dxForm/1 Configuration/alignItemLabels.md (hunk 3 above)
Comment (novsstation) [reply api-reference/10 UI Components/dxForm/1 Configuration/alignItemLabels.md]: After implementing floating labels, there are 2 types of labels: form.labels and editors.labels. So, customers may confuse. May be it's better to specify, that this option affects only form.labels, not editor.labels?

Is the variant below more clear?
"Applies only if the labels are rendered as the form labels, not editor labels ([labelMode](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/labelMode.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode') is *"outside"* or [visible](api-reference\10%20UI%20Components\dxForm\5%20Item%20Types\SimpleItem\label\visible.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#visible') is set to a true for the form labels)."

Is int necessary to specify the text above in all topics with "applies only" text?

File: api-reference/10 UI Components/dxForm/1 Configuration/labelMode.md [hunk 5]
```diff
@@ -25,4 +25,21 @@ Specifies a display mode for [item labels](/api-reference/10%20UI%20Components/d
...
     </tr>'
-}
\ No newline at end of file
+}
+
+This property specifies a display mode for all item labels. If you want to override the mode for an individual label, specify the **labelMode** property within [editorOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions).
+
+[note]
+
+The following [editors](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) do not support the *"static"* and *"floating"* modes and will fall back on the *"outside"* mode:
```
Comment (arminal) [inline api-reference/10 UI Components/dxForm/1 Configuration/labelMode.md]:
```suggestion
The following [editors](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) do not support *"static"* and *"floating"* modes and will fall back on the *"outside"* mode:
```

File: api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/visible.md [hunk 6]
```diff
@@ -5,6 +5,12 @@ default: true
...
 ---
 ##### shortDescription
-Specifies whether or not the label is visible.
+Controls the visibility of the label outside the editor.
 
 ---
+This property defaults to **true** in the following cases:
+
+- The [labelMode](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode) is *"outside"*.
+- The editor does not support embedded labels and therefore does not support the *"static"* and *"floating"* label modes. Refer to the [labelMode](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode) article for a list of these editors.
```
Comment (arminal) [inline api-reference/10 UI Components/dxForm/5 Item Types/SimpleItem/label/visible.md]:
```suggestion
- The editor does not support embedded labels and therefore does not support *"static"* and *"floating"* label modes. Refer to the [labelMode](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#labelMode) article for a list of these editors.
```

**Comment by arminal** [comment]
#GrammarReview

Good job, I only edited your article usage with the word "mode".

**Pull request merged**